src/applications/personalization/dashboard-2/components/health-care/HealthCare.jsx

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import NotificationCTA from '../NotificationCTA';
 
+import { isAuthenticatedWithSSOe } from 'platform/user/authentication/selectors';
+import { mhvUrl } from 'platform/site-wide/mhv/utilities';
+// import { recordDashboardClick } from 'applications/personalization/dashboard/helpers';
```

Comment:
Just commented this out for now - I think we'll need to record the click on `health-care/get-medical-records` like we do now in the current Dashboard.